packages/docs/docs/concepts/cornerstone-tools/tools.md

```diff
@@ -164,22 +164,25 @@ interactions.
 Contour tools like the PlanarFreehandROITool can be configured can be configured to use interpolation process.
 The interpolation process uses b-spline algorithm and consider 3 configurations properties:
 
-- interpolation.enabled: to tell whether it should be interpolated or not (false by default)
-- interpolation.minKnotDistance: the minimum distance between each knot on interpolation process (for editing it is considered the property editMinKnotDistance) ( default value 20)
-- interpolation.editMinKnotDistance: same as minKnotDistance but applicable only when editing the tool (default value 10)
+- interpolation.enabled: to tell whether it should be interpolated or not (for editing it is considered the property editEnabled) (default: false)
+- interpolation.editEnabled: to tell whether it should be interpolated or not when editing (default: false)
+- interpolation.knotsRatioPct: percentage of points from Segment that are likely to be considered knots during interpolation (for editing it is considered the property editKnotsRatioPct) ( default: 40)
+- interpolation.editKnotsRatioPct: same as knotsRatioPct but applicable only when editing the tool (default: 40)
+- interpolation.forceAllSegment: tell whether interpolation occurs on edited segment only or all drawing (true means all drawing) (default: false)
```

Comment:
Hmmm, I don't think this is the right place to explain about all of this options. This page is a very high level concept of tools, and what is there to use rather than tool specific options. I even don't like the fact that we have `Tool interpolation` here (it was merged when I was out I guess). Can we keep this place as high level as possible, and you comment in the Contour tool class so that it appears in the API docs? Similar to here https://www.cornerstonejs.org/api/tools/class/WindowLevelTool for windowLevel tool.


Also please remove the `Tool Interpolation` and `how to interpolate` section from here. Where to put you might ask, I think the API doc is enough. We _can_ put a subsection (at the level of annotation tools) called contour tools, and just mention we have contour tools.

Reply:
Actually we have it here https://www.cornerstonejs.org/api/tools/class/planarfreehandroitool/ there is some doc, just move these two sections there